resources.whatwg.org/build/deploy.sh

```diff
@@ -63,6 +63,12 @@ echo ""
 
 rm -rf "$WEB_ROOT" || exit 0
 
+copy_extra_files() {
+    if [[ "$EXTRA_FILES" != "" ]]; then
+        cp $EXTRA_FILES $1
```

Comment:
It appears we're not running shellcheck on this script. (It does not like the grep regexp also.)

Per https://github.com/koalaman/shellcheck/wiki/SC2086, it suggests we quote at least the `$1`. It also wants us to quote the `$EXTRA_FILES`, but that defeats the point; I've asked about what the right pattern is at https://stackoverflow.com/q/45931553/3191 .

In the meantime, let's just quote `$1` at least.

Reply:
Ah, thanks, wonder what the answer will be :)

Reply:
Also adding a link to the question in the source.